packages/wk-lib-playground/package.json

```diff
@@ -75,7 +74,8 @@
   },
   "peerDependencies": {
     "react": "^17.0.2",
-    "react-dom": "^17.0.2"
+    "react-dom": "^17.0.2",
+    "styled-components": "^5.3.3"
```

Comment:
styled-components should be a `peerDependency` in your library like react and react-dom (to indicate to consumer libraries that it needs to be installed, but so that you don't have to package the entire styled-components project in your custom library)